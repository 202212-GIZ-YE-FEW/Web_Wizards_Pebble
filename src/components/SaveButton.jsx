const SaveButton = ({ text, className }) => {
    return (
        <div className='px-3 py-2 lg:py-4 '>
            <button
                className={`w-36 h-16 lg:w-56 bg-primary-200  text-white font-semibold rounded-lg text-sm lg:text-md px-3 sm:px-5 py-3  ${className}`}
            >
                {text}
            </button>
        </div>
    );
};

export default SaveButton;
