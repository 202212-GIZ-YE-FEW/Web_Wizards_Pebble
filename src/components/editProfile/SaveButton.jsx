import { Button } from "react-flatifycss";

const SaveButton = ({ text, className }) => {
    return (
        <Button
            className={`!w-36 !h-16 lg:!w-56 !bg-primary-200  !text-white !font-semibold !rounded-lg !text-sm lg:!text-md !px-3 sm:!px-5 !py-3 !shadow-none !border-none ${className}`}
        >
            {text}
        </Button>
    );
};

export default SaveButton;
