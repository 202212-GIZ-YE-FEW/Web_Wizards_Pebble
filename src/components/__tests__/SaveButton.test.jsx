import renderer from "react-test-renderer";

import SaveButton from "@/components/editProfile/SaveButton";

it("renders correctly", () => {
    const tree = renderer.create(<SaveButton />).toJSON();
    expect(tree).toMatchSnapshot();
});
