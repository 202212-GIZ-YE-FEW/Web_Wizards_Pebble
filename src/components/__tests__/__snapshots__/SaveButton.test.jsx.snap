// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="px-3 py-2 lg:py-4 "
>
  <button
    className="w-36 h-16 lg:w-56 bg-primary-200  text-white font-semibold rounded-lg text-sm lg:text-md px-3 sm:px-5 py-3  undefined"
  />
</div>
`;
